docs/dev/libvirt/README.md

```diff
@@ -23,7 +23,7 @@ crw-rw-rw-+ 1 root kvm 10, 232 Oct 31 09:22 /dev/kvm
 If it is missing, try some of the ideas [here][kvm-install].
 
 ### Install and Enable Libvirt
-On CentOS 7, first enable the
+On CentOS/RHEL 7.x, first enable the
 [kvm-common](http://mirror.centos.org/centos/7/virt/x86_64/kvm-common/)
```

Comment:
On RHEL7, it really depends, if you have access to the RH virt channel, it's best to use that.

Reply:
I am afraid, if that has the latest stuff. Do you think otherwise?

Reply:
base RHEL7 has an old qemu version, but there are more recent virt packages available when you have an openstack/rhv subscription, which gives you access to `qemu-kvm-rhev` which has a 2.12 qemu version

Reply:
Ahh, I see. As far as this PR is concerned, some pointer around this will be helpful to the users in general. So, let me know, if you think adding a RHEL specific note will be helpful around this. Or if you like, you can take over and do the needful.

Reply:
It would be best to describe the "if you have access to the RH virt channel, it's best to use that." as this is already documented.

Reply:
It's all related to old version of qemu-kvm that RHEL is using. As @gbraad said it's not available in RHEL7, only RHEL8 or you need to enable Advanced Virt repository on RHEL 7, it also comes in RHV/OSP repositories. Then you can replace the old version with new qemu-kvm-rhev. Hope it helps @ScrapCodes

Reply:
https://github.com/qemu/qemu/commit/81b2b81062612ebeac4cd5333a3b15c7d79a5a3d - The patch that added **-fw_cfg** option to qemu-2.4, to be able to use it on RHEL7 you will need to enable **rhel-7-server-rhv-4-mgmt-agent-rpms** repository.

Reply:
Thanks, for the full information on this, definitely, it helps.